octoprint_TpLinkAutoShutdown/__init__.py

```diff
@@ -140,6 +160,9 @@ def get_settings_defaults(self):
 				deviceOne=True,
 				deviceTwo=True,
 				deviceThree=True,
+    			deviceFour=True,
```

Comment:
Don't know if this is just the way I am viewing the code on GitHub but this looks to be indented incorrectly. Could you please have a look as deviceFour to me looks as if it needs to be indented again to conform the the rules.